content/docs/quickstart/sqlalchemy.md

```diff
@@ -2,13 +2,23 @@
 title: Run a SQL Alchemy App
 ---
 
+### Introduction
+
 SQLAlchemy is among the most popular ORMs in the Python universe. Because Neon is fully compatible with vanilla PostgreSQL, you only need to fill in the correct connection details.
 
-Prerequisites:
+## Prerequisites
+
+To complete this section, you will need:
+
+- Already created a Project on Neon Cloud service
+- Have SQLalchemy installed with a PostgresSQL driver (such as psycopg2, the default driver for PostgreSQL in SQLAlchemy)
```

Comment:
SQLAlchemy

Reply:
Fixed in this commit, https://github.com/neondatabase/website/pull/59/commits/6df68c70e4f0a8ed7cd739bec9d5aa5c4456a795

Good catch, thank you!